packages/flutter/lib/src/material/text_form_field.dart

```diff
@@ -332,7 +332,7 @@ class TextFormField extends FormField<String> {
     BuildContext context,
     EditableTextState editableTextState,
   ) {
-    if (defaultTargetPlatform == TargetPlatform.iOS && SystemContextMenu.isSupported(context)) {
```

Comment:
Looks like analyzer is complaining we no longer need the foundation import `package:flutter/foundation.dart` in this file.